src/components/AccountSwitcher.tsx

```diff
@@ -87,34 +86,33 @@ function AccountSwitcher() {
 
             return [
                 createBaseMenuItem(delegatePersonalDetails, error, {
-                    onPress: () => {
+                    onSelected: () => {
                         if (isOffline) {
                             Modal.close(() => setShouldShowOfflineModal(true));
                             return;
                         }
                         disconnect();
                     },
-                    key: `${delegateEmail}-delegate`,
+                    isSelected: true,
```

Comment:
Why is this needed now? And what about the `currentUserMenuItem`? Will it still be focused?

Reply:
That is my mistake, we should add `isSelected` as `true` for `currentUserMenuItem`.